### DISCUSSION TITLE
"Unfortunately, we weren't able to approve your educational discount request"

### DISCUSSION THREAD

**Discussion by AliAmmarNaseem** [community/community#52884] [GitHub Education]
I have been trying to apply for the student developer pack for many days now, but unfortunately, I have been rejected 20 times after uploading an image. I have tried uploading different documents like my student ID card, LMS screenshot, fee challan and semester schedule but still not getting approval.

I am attaching a real image issued by my institute which is not blurry and still facing problems with approval. 

**When I upload from gallery, they say :**
"The item you uploaded is insufficient to demonstrate your current academic status.
Please see suggestions.
Suggested proof to expedite your approval:
Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy."
 
**When I use device camera and capture image then get following error:**
"Unfortunately, we weren't able to approve your educational discount request for {my username}

What happened?
Common verification problems and suggested solutions are documented here: https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved."

**Things I have tried:**

- Image of all documents are legible and texts are clear and easily readable. 
- Image taken is of larger resolution.
- Tried different type of documents like Student ID Card, Result card, Fee challan, Semester time table
- All Documents are in English.
- Name, username of my GitHub profile as same as my University ID card and other documents.
- My profile is set to public.

What should I do now? Please help me!

Thank you for your time and consideration.

![Screenshot 2023-04-06 175701](https://user-images.githubusercontent.com/114722213/230385500-fd8579d7-48b3-440c-a136-36635f6406bb.png)
![Screenshot 2023-04-06 175720](https://user-images.githubusercontent.com/114722213/230385510-e37162ed-7988-450c-91c3-d3546d340999.png)

**Comment by Ntshangase** [comment]
Hey @AliAmmarNaseem 

If you recently updated your GitHub profile I would suggest that you logoff and login to GitHub again then proceeded to apply again. Also make sure GitHub can pickup your location.

**Reply by AliAmmarNaseem** [reply]
I have already tried that multiple times and its been a month and still facing this problem even after 40 requests. Please do it manually.

**Reply by Ntshangase** [reply]
I'm not part of the support team, I was just trying to assist.

**Comment by samus-aran** [comment]
Hi @AliAmmarNaseem 

The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your discussion was locked and closed because it contained content that violates our Community Code of Conduct, including posting the same comment/discussion in multiple places. [This is original discussion that supersedes this](https://github.com/orgs/community/discussions/52169), and we ask that you no longer cross-post.

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).
